src/services/user.ts

```diff
@@ -257,6 +264,9 @@ const setMobilePhonePrivacy = (makePrivate: boolean) =>
 const setHomePhonePrivacy = (makePrivate: boolean) =>
   http.put('profiles/mobile_privacy/' + (makePrivate ? 'Y' : 'N')); // 'Y' = private, 'N' = public
 
+const setPlannedGraduationYearPrivate = (makePrivate: boolean) =>
+  http.put('profiles/planned_graduation_year_privacy/' + (makePrivate ? 'Y' : 'N')); // 'Y' = private, 'N' = public
```

Comment:
```suggestion
  http.put('profiles/planned_graduation_year_privacy/' + (makePrivate ? 'Y' : 'N')); // 'Y' = private, 'N' = public
```
This was something I mentioned in the API PR: why are we switching between integers, strings, and booleans. We should be consistent?

Reply:
changed so it will just pass a boolean instead of a string